server/config.py

```diff
@@ -4,11 +4,12 @@
 """
 DEBUG = False
 
+# TODO these are hard coded configurations and should be loaded from a resource provider
+
 SERVICE_ROOT = 'https://cida.usgs.gov'
 STATS_SERVICE_ROOT = 'http://cida.usgs.gov/ngwmn_statistics'
 
-# TODO this will be revised to be loaded from configuration
-CONFLUENCE_URL = 'http://ngwmn-dev.cr.usgs.gov:8080/ngwmn-remote-content/content/providers/'
+CONFLUENCE_URL = 'http://ngwmn-prod.cr.usgs.gov:8080/ngwmn-remote-content/content/'
```

Comment:
FYI: It has been our practice to not put internal URLS in github repos

Reply:
oh, thanks for the heads up. I did not know that. Make sense however. 
I do not think this new service has an external URL.
We should probably pull this project into gitlab like everything else.
I placed a comment in the file that we should be loading these from a service.

Reply:
Yeah, I agree.  It does not seem like a huge security risk, but not good practice either.
Ultimately, this change is intended to be temporary until we fix configuration for this app:  Right now the configuration is in the repo and we need it to be actually configurable.